health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/PaymentsCard.js

```
@@ -0,0 +1,43 @@
+  const { t } = useTranslation();
+  const userInfo = Digit.UserService.getUser();
+  const userRoles = userInfo?.info?.roles?.map((roleData) => roleData?.code);
+  const generateLink = (labelKey, pathSuffix,roles=ROLES.ATTENDANCE) => {
+    return {
+      label: t(labelKey),
+      link: `/${window?.contextPath}/employee/payments/${pathSuffix}`,
+      roles: roles,
+    };
+  };
+  
+  if (!Digit.Utils.didEmployeeHasAtleastOneRole(Object.values(ROLES).flatMap((e) => e))) {
```

Biome (1.9.4)

[error] Avoid unnecessary callback in flatMap call.

You can just use flat to flatten the array.
Safe fix: Replace unnecessary flatMap call to flat instead.


(lint/correctness/noFlatMapIdentity)